pkg/virt-launcher/virtwrap/live-migration-source.go

```diff
@@ -399,8 +400,10 @@ func (l *LibvirtDomainManager) setMigrationResultHelper(failed bool, completed b
 		}
 		if completed {
 			migrationMetadata.Completed = true
-			now := metav1.Now()
-			migrationMetadata.EndTimestamp = &now
+		}
+
+		if string(abortStatus) != string(v1.MigrationAbortInProgress) {
+			migrationMetadata.EndTimestamp = pointer.P(metav1.Now())
```

Comment:
Abortion can also fail. In what case we need end timestamp and not completed?

Reply:
This is an abortion in progress though which is not the end of a migration, rather we're waiting for the abortion request to be processed so in this case I think we should not be setting the end timestamp.

Reply:
What I mean here is that you set the `EndTimestamp` even if the abortion fails. This seems wrong. Maybe we can add unit test?

Reply:
I think we should, the migration process has ended, even if it was aborted and the abortion failed. No?

Reply:
It might not be aborted and you will see abortion failed.

Reply:
Oh I see what you mean, `setMigrationAbortStatus` currently sets Failed to false while I though it'd set it to True, then yes it makes more sense to not set the EndTimestamp when the migration gets aborted.

Reply:
Just re-pushed, now abortion never sets the endTimestamp.